### DISCUSSION TITLE
Elementary OS 8 Login Issues

### DISCUSSION THREAD

**Discussion by Cotywio** [elementary/triage#481] [Q&A]
Hello,

Apologies as I'm not sure which repo I should report this in, upon downloading the Elementary OS 8 Early Beta (at least March 7th - 12th) I am having the following issues:

1. When I click into "Try Demo Mode" I get the following message:
        <img src="https://github.com/elementary/triage/assets/17916912/e5e1daef-1376-4f4f-9986-264a0a1373e5" width="600">

2. When I install via the option to wipe the entire disk, with no encryption and with/without third-party drivers, I'm able to get here:
       <img src="https://github.com/elementary/triage/assets/17916912/744b53ef-fa7a-4a24-a7e8-cc116f2987cb" width="600">

3. If I type in my password, it brings me to this screen, all black except for the panel icons. If I move my mouse to hover over the icons, they disappear and I can no longer see anything. 
       <img src="https://github.com/elementary/triage/assets/17916912/b43b337e-576c-4c36-be5b-076e9ccc1ff4" width="600">

   Not sure if this is helpful, but including .xsession-errors found in my user profile:
a.
        <img src="https://github.com/elementary/triage/assets/17916912/0b950256-197b-4467-b1dc-2e659faa9ed4" width="600">

If I try via Pantheon (Wayland), I receive a completely black screen with no panel icons at all. 

 
Platforms tested on (booting in EFI mode): 
-   VirtualBox
-   Dell Inspiron Laptop
-   Dell Latitude 3410 Laptop
Media created with Rufus & BelenaEtcher with similar errors for both. 

Thank you!

**Comment by blondejefe** [comment]
I too have this problem and I've noticed others at Reddit hoping for a solution also. That said, isn't "DISPLAY=:0.0" the correct environment variable?

**Comment by lenemter** [comment]
Switch to tty1 (Ctrl+Alt+F1) enter your username and password, then execute `coredumpctl debug gala > backtrace.txt` and upload `backtrace.txt` here please

**Reply by Cotywio** [reply]
Ran this and I'm receiving a "No match found." message with no data in backtrace.txt

**Reply by lenemter** [reply]
Hm... Then can you please debug it manually?

1. Switch to tty1 by pressing Ctrl+Alt+F1
2. Install gdb (`sudo apt install gdb`)
3. Run `DISPLAY=:0 gdb gala`
4. In gdb prompt run `set logging on`
5. Then `run -r --display=:0`
6. Gala will crash
7. Run `bt`


After all of this please send the gdb.txt file from your home directory

**Reply by Cotywio** [reply]
Received some output there, attached! Thanks for the help with this. :) 
[gdb.txt](https://github.com/elementary/triage/files/14591399/gdb.txt)

**Reply by tnetenbaa** [reply]
I put something about this out on Mastodon, and the official Elementary page responded, apparently something is broken on the Ubuntu end as of now.

**Comment by blondejefe** [comment]
Can anyone confirm if this has been fixed yet?

**Reply by tnetenbaa** [reply]
Not entirely sure, I downloaded the ISO as of two days ago, and couldn't get past an initial black screen on a few different systems.

**Comment by Trishun** [comment]
Same here, I've installed as clean install ~1 month ago

**Comment by Trishun** [comment]
OK, I've probably just find workaround:
```
as root create file /etc/X11/Xsession.d/01custom-has-option with following content:

OPTIONS="$(
  if [ -r "$OPTIONFILE" ]; then
    cat "$OPTIONFILE"
  fi
  if [ -d /etc/X11/Xsession.options.d ]; then
    run-parts --list --regex '\.conf$' /etc/X11/Xsession.options.d | xargs -d '\n' cat
  fi
)"

#check if has_option is already defined; if not, define

has_option 2>1 > /dev/null || has_option() {
  # Ensure that a later no-foo overrides an earlier foo
  if [ "$(echo "$OPTIONS" | grep -Eo "^(no-)?$1\>" | tail -n 1)" = "$1" ]; then
    return 0
  else
    return 1
  fi
}
```
Then reboot.
Source: https://bugs.launchpad.net/ubuntu/+source/xorg/+bug/1955135/comments/9

I've suspended my laptop several times now and by now problem didn't occur.
